### DISCUSSION TITLE
Adding directories?

### DISCUSSION THREAD

**Discussion by iamericfletcher** [tldr-pages/tldr#9670] [Ideas]
What are your thoughts on adding overviews for the directories that comprise the Linux filesystem hierarchy?

Fo example, a command for `/etc` could looks something like this:

 `tldr /etc`

Output:

`The /etc hierarchy contains configuration files. A "configuration file" is a local file used to control the
operation of a program; it must be static and cannot be an executable binary. 2
It is recommended that files be stored in subdirectories of /etc rather than directly in /etc.`

Taken from - https://refspecs.linuxfoundation.org/FHS_3.0/fhs-3.0.pdf

**Comment by CleanMachine1** [comment]
An interesting concept for the project. Personally I feel its out of the reach of the project.


All pages must follow the same format of 

```
# title

> Description
> More information link

- command example:

`command`
```
I don't see how this could be done with our format of pages, do you or anyone reading have any ideas?

**Comment by EmilyGraceSeville7cf** [comment]
> What are your thoughts on adding overviews for the directories that comprise the Linux filesystem hierarchy?

I really appreciate this idea. Instead of command description we can write directory one. And instead of code examples we can write about the common files or directories present in this directory like that:

```md
- Bla bla bla:

`file-name`
```

Not to introduce conflicts with existing pages we can use trailing slash for directory pages.
